src/style.css

```
@@ -1 +1,66 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin-top: 22.5px;
+  margin-bottom: 22.5px;
+}
+
+.nav__element {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+}
+
+.nav__element:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:hover,
+.is-active {
+  color: var(--main-color);
+
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 33px;
```

```suggestion
  bottom: 0;
```
задай на лінк position: relative;
щоб позиціонувалась ця штука відносно лінки, а не всього хедера